include/defines.h

```diff
@@ -308,4 +308,12 @@
  */
 #define KMYTH_GETKEY_RX_BUFFER_SIZE 256
 
+/**
+ * 
+ * Length of hexadecimal string of the policy digest read within TPM 2.0.
+ * (in characters).
+ *
+ */
+#define POLICY_DIGEST_HEXSTRING_SIZE 132
```

Comment:
I think this number is based on incorrect logic. I think that the hex size for a policy digest should be twice the size of the number of hash bytes produced by the algorithm used. For SHA-256, for example, which produces a 32-byte hash result, I think the hex string should be 64 bytes (plus a string terminator byte, perhaps).